### PR TITLE
LPS 197679 Show missing labels in DPT's infopanel and formatting

### PR DESCRIPTION
**Pull request by YurenaC**
Motivation
=======
When reviewing the following [figma](https://www.figma.com/file/ydHldCxSGZuPe1tuHu9jXe/LPS-189856---Usability-improvements-for-objects-page-builder-integration?type=design&node-id=1-453&mode=design&t=ZOGpD9JjHbOKG9IM-0), we realized that some data and formatting was missing.

Proposed Solution
========
Implement the missing data and formatting when the data is shown.

Steps to Verify
========
1. Go to Design > Page Templates > Display Page Templates Tab.
2. Create a new Folder.
3. Create a new Display Page Template.
4.  Check that the styles and data showed in the infopanels matched the examples given in the previously linked figma file.

Commits
========
- LPS-197679 Show status and default label when corresponds in LayoutPageTemplateEntry info-panel
- LPS-197679 Unifying info-panel styles
- LPS-197679 Unifying info-panel date styles
- LPS-197679 Hiding description when empty
- LPS-197679 Add missing language key
- LPS-197679 Add outer div so it gets proper height
- LPS-197679 Fix spacing and text style
- LPS-197679 buildLang

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/display/context/DisplayPageTemplateInfoPanelDisplayContext.java [hunk 1]
```diff
@@ -55,6 +60,32 @@ public DisplayPageTemplateInfoPanelDisplayContext(
 			WebKeys.THEME_DISPLAY);
 	}
 
+	public String getCreationDateAndAuthor(
```
Comment (jkappler) [inline modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/display/context/DisplayPageTemplateInfoPanelDisplayContext.java:63]: @YurenaC, we already have a pattern for this, please check info_panel_file_entry.jsp for example

File: modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/display/context/DisplayPageTemplateInfoPanelDisplayContext.java [hunk 2]
```diff
@@ -155,6 +186,27 @@ public List<LayoutPageTemplateEntry> getLayoutPageTemplateEntries() {
 		return _layoutPageTemplateEntries;
 	}
 
+	public String getModificationDate(
```
Comment (jkappler) [inline modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/display/context/DisplayPageTemplateInfoPanelDisplayContext.java:189]: @YurenaC, same for this in other info panels, for example in the one for journal

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**